app/javascript/app/pages/home/sections-slideshow/sections-slideshow-component.jsx

```diff
@@ -1,48 +1,115 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Button, Carousel } from 'cw-components';
-import ghgImage from 'assets/historical-emissions@2x';
+import Link from 'redux-first-router-link';
+// redirect actions
+import { NATIONAL_CONTEXT, CLIMATE_GOALS } from 'router';
+// images
+import climateSmImage from 'assets/carousel_climate_goals@1x';
+import climateBgImage from 'assets/carousel_climate_goals@2x';
+import nationalSmImage from 'assets/carousel_national_context@1x';
+import nationalBgImage from 'assets/carousel_national_context@2x';
+import anualSmImage from 'assets/carousel_annual_emissions@1x';
+import anualBgImage from 'assets/carousel_annual_emissions@2x';
+import provincesSmImage from 'assets/carousel_top_10_emitting_provinces@1x';
+import provincesBgImage from 'assets/carousel_top_10_emitting_provinces@2x';
 
 import styles from './sections-slideshow-styles.scss';
 
-const pagingTitles = [
-  'Climate goals',
-  'National context',
-  'Annual emissions',
-  'Top 10 emitting provinces'
+const slidesData = [
+  {
+    pagingTitle: 'Climate goals',
+    title: 'Indonesia’s climmate commitment',
+    buttonText: 'Go to climate goals',
+    smImage: climateSmImage,
+    bgImage: climateBgImage,
+    altText: 'Climate goals chart',
+    redirectAction: CLIMATE_GOALS
+  },
+  {
+    pagingTitle: 'National context',
+    title: 'Socioeconomic indicators',
+    buttonText: 'Go to national context',
+    smImage: nationalSmImage,
+    bgImage: nationalBgImage,
+    altText: 'National context chart',
+    redirectAction: NATIONAL_CONTEXT
+  },
+  {
+    pagingTitle: 'Annual emissions',
+    title: 'Greenhouse Gas Emissions and Targets',
+    buttonText: 'Go to GHG target',
+    smImage: anualSmImage,
+    bgImage: anualBgImage,
+    altText: 'GHG chart',
+    redirectAction: CLIMATE_GOALS
+  },
+  {
+    pagingTitle: 'Top 10 emitting provinces',
+    title: 'Top 10 emitting provinces',
+    buttonText: 'Go to historical emissions',
+    smImage: provincesSmImage,
+    bgImage: provincesBgImage,
+    altText: 'Provinces chart',
+    redirectAction: CLIMATE_GOALS
+  }
```

Comment:
do we know if this data will be coming from data files or it's gonna be manageable from admin panel? If so, we can wait until we merge translations branch and we add slugs for these to fetch content from BE

Reply:
Not really sure about this one, perhaps @simaob knows better.

Reply:
Doesn't look like sth coming from data files. Translations are already on develop, so now you can rebase and rely on content coming from BE :) more info here ---> https://github.com/ClimateWatch-Vizzuality/indonesia-platform/pull/17 
or just ping me when in doubts :D